### DISCUSSION TITLE
ajuda com git.

### DISCUSSION THREAD

**Discussion by CaioCezarDeOlivera2023** [community/community#74967] [Repositories]
### Select Topic Area

Question

### Body

ola, gostaria saber se alguém pode me ajudar, pois estou tentando subir uma commit, porém sempre pede usuário e senha, já redefini a senha,já reconfigurei a chave ssh, ja criei uma chave tokem e ja até reinstalei o git, porém mesmo assim da falha de autenticação, podem me ajudar.

**Comment by Akash1134** [comment]
Hi @CaioCezarDeOlivera2023 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
